db/seeds.rb

```diff
@@ -1 +1 @@
-AdminUser.create!(email: 'admin@example.com', password: 'password')
+AdminUser.create!(email: 'admin@example.com', password: 'password') if Rails.env.development?
```

Comment:
Does it make any sense to keep on seeding in every test then?
Maybe we should change the approach we are using on our projects to make tests faster. Instead of always seeding before a test, invoking the seeds on the tests where we need them, thoughts?

Reply:
I'm not sure if we are seeding before every test 🤔

Reply:
You are right, I checked and we are not. Got confused because I saw it in many projects 👍